doc/guides/pm/power_domain.rst

```diff
@@ -4,21 +4,20 @@ Power Domain
 Introduction
 ************
 
-Most of the devices in an SoC have independent power control that can
-be turned on or off to reduce power consumption. But there is a
-significant amount of static current leakage that can't be controlled
-only using device power management. To solve this problem, SoCs
-normally are divided into several regions grouping devices that
-are generally used together, so that an unused region can be
-completely powered off to eliminate this leakage. These regions are
-called "power domains", can be present in a hierarchy and can be
-nested.
+The Zephyr power domain abstraction is designed to support groupings of devices
+powered by a common source to be notified of power source state changes in a
+generic fashion. Application code that is using device A does not need to know
+that device B is on the same power domain and should also be configured into a
+low power state.
```

Comment:
I am finding this very last sentence confuse (or not necessary). The "and should also be configured into a low power state".